server/integration/endpoint/src/test/java/org/infinispan/server/endpoint/EndpointSubsystemTestCase.java

```diff
@@ -69,10 +69,10 @@ public EndpointSubsystemTestCase(String xmlFile, int operations, String xsdPath,
    @Parameters
    public static Collection<Object[]> data() {
       Object[][] data = new Object[][] {
-            { "endpoint-7.2.xml", 16, "schema/jboss-infinispan-endpoint_7_2.xsd", null },
-            { "endpoint-8.0.xml", 16, "schema/jboss-infinispan-endpoint_8_0.xsd", null },
-            { "endpoint-9.0.xml", 38, "schema/jboss-infinispan-endpoint_9_0.xsd", null },
-            { "endpoint-9.2.xml", 38, "schema/jboss-infinispan-endpoint_9_2.xsd", new String[] { "/subsystem-templates/infinispan-endpoint.xml"} },
+            { "endpoint-7.2.xml", 15, "schema/jboss-infinispan-endpoint_7_2.xsd", null },
+            { "endpoint-8.0.xml", 15, "schema/jboss-infinispan-endpoint_8_0.xsd", null },
+            { "endpoint-9.0.xml", 37, "schema/jboss-infinispan-endpoint_9_0.xsd", null },
```

Comment:
In my understanding, the old config files and old xsds didn't change, so this count should remain the same?

Reply:
The count is the number of executed operations, so removing the "com.sun.security.sasl.digest.utf8" property means that the count must be decremented by one.

Reply:
Got it, thanks